### DISCUSSION TITLE
4.6.22版本没有直播分区选项，礼物无法发送

### DISCUSSION THREAD

**Discussion by TsubakiAwa** [ywmoyue/biliuwp-lite#422] [bug]
### Bug 描述

![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/9d925a67-73ee-44b6-84f7-892e0f45727c)

![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/fa3c827a-475c-446f-8bfa-acd71f211c75)
![2](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/12eabc47-4dc9-49ee-8210-dc91db6ca311)

### 操作步骤

```plain text
我记得以前版本直播版块是有直播分区的，但是上个版本好像是4.6.20就出现无法正常显示打开了，然后4.6.22修复后少了个直播分区选项,弹幕上个版本4.6.21还能正常发送弹幕但是4.6.22更新后弹幕就无法发送了不知道是不是有延迟显示，礼物好像以前就一直发生不了只能看，发送还得用客户端或者网页才行
```

### 期望结果

![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/06c64a23-470e-475b-8d56-7436ba959b53)

### 日志

[20231205.log](https://github.com/ywmoyue/biliuwp-lite/files/13556527/20231205.log)

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
初步看了一下，没有直播分区应该是b站api改了，现在不返回相关值`area_entrance_v2`。其他的弹幕和礼物问题请看看等会出的新版本是否有解决。@TsubakiAwa

**Reply by TsubakiAwa** [reply]
弹幕好像还是一样，可能我没说清楚，弹幕的问题是自己发送的看不见能看见别人的让我错以为的没有发送出来，但是在别的客户端比如网页或者移动端都能看见自己刚发送出来的弹幕
![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/869863ad-5d29-4ce3-a4f1-dc04993b267b)

**Reply by GD-Slime** [reply]
emmm我这里无法复现，我在多个直播间都发了弹幕，自己的弹幕是可以被看见的。
你的意思是，在biliuwp-lite中发送弹幕, biliuwp-lite中看不到自己弹幕，但是其他终端是可以看到的，是这样吗？
你使用的是什么方式登录呢？我使用扫码登录，也许是登录方式导致的？

**Reply by TsubakiAwa** [reply]
也是扫码登录，但我感觉像是常用在biliuwp-lite里登录的账号被封控了，我换了主号就能看见了不好意思打扰到你了
![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/b2902736-e14d-454e-8a9b-bf1214681be3)

**Reply by GD-Slime** [reply]
@TsubakiAwa 找到问题了。你是否账号没有佩戴粉丝牌发送弹幕？这是个bug, 已经在 #435 修了

**Comment by ywmoyue** [comment]
@TsubakiAwa 试试4.6.23有没有解决

**Comment by GD-Slime** [comment]
@TsubakiAwa 请问你 **期望结果** 中的这个界面是哪个b站客户端版本呢？ 我这里安卓手机端和ipad端均获取不到这个直播分区的卡片，只有上面的`banner`和下面的`我的关注`. 
目前我这里抓包抓到的和原有的直播分区相似的功能只有文字，没有配套的图片，套用不到原来的页面框架上。

**Reply by TsubakiAwa** [reply]
粉色的哔哩哔哩正式版app改了但是先从酷安下载[哔哩哔哩概念版](https://www.coolapk.com/apk/com.bilibili.app.blue)然后再从app里更新到7.56.0直播页面还是旧版的
![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/f5730642-f175-445f-9334-d8e94901397f)
![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/adcf70ae-399c-4f43-aee1-f096815bb20c)

**Reply by GD-Slime** [reply]
ipad概念版7.58版本已经把这个移除了。我试试降级或者从别的平台尝试抓包吧。

**Reply by ywmoyue** [reply]
@TsubakiAwa 试试短信登录后查看有没有这个直播分区

**Reply by GD-Slime** [reply]
@ywmoyue 我这里拿短信登录后就有这个直播分区了

**Reply by TsubakiAwa** [reply]
短信登录一直提示我验证码错误，没办法试，但我用网页端是可以短信正常登录的

![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/6273d782-0114-46a8-ae04-3c047e00a1eb)
![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/105eb8b8-debd-4d15-804d-8a66d15462cf)
![2](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/8a2757bd-902c-43d3-8758-d75d8a7f1940)

**Reply by ywmoyue** [reply]
这个流程缺了一些关键日志，很难定位到问题，下个版本补上

@TsubakiAwa  以及问下，你安装uwp的架构是x86还是arm

**Reply by TsubakiAwa** [reply]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/fdaebe8a-9d26-4431-b346-d49fed1ce2a7)

![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/0df88e22-94d2-4311-b9f7-4340c05955ab)

![image](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/44371a33-46d1-45c8-afb1-12016c4d55df)
安装的是X86的，
[20231221.log](https://github.com/ywmoyue/biliuwp-lite/files/13735281/20231221.log)

**Comment by ywmoyue** [comment]
@TsubakiAwa 麻烦验证下4.6.24是否已解决这个问题

**Reply by TsubakiAwa** [reply]
直播分区啥的都好了，但是短信验证码登录还是会出现验证码错误，我还是用扫码登录吧，如果还能修的话就修，不能就算了
![4](https://github.com/ywmoyue/biliuwp-lite/assets/53134066/bf4b38fb-50eb-4927-8028-aab106c5c7dd)
[20231224.log](https://github.com/ywmoyue/biliuwp-lite/files/13761824/20231224.log)
